app/code/Magento/Sales/view/frontend/templates/email/items/shipment/default.phtml

```diff
@@ -15,7 +15,14 @@ $_item = $block->getItem() ?>
                 <?php foreach ($block->getItemOptions() as $option) : ?>
                     <dt><strong><em><?= $block->escapeHtml($option['label']) ?></em></strong></dt>
                     <dd>
-                        <?= /* @noEscape */  nl2br($option['value']) ?>
+                        <?php
+                        if ($option['option_type'] == 'file') {
```

Comment:
Should be \Magento\Catalog\Api\Data\ProductCustomOptionInterface::OPTION_TYPE_FILE

Reply:
I have applied changes as per suggestion as well as I have tried and fixed some phpcs errors.